presto-tests/src/test/java/com/facebook/presto/tests/TestTransactionManagerIntegration.java

```diff
@@ -13,31 +13,63 @@
  */
 package com.facebook.presto.tests;
 
+import com.facebook.airlift.log.Logger;
 import com.facebook.presto.tests.tpch.TpchQueryRunnerBuilder;
+import com.facebook.presto.transaction.TransactionInfo;
+import com.facebook.presto.transaction.TransactionManager;
 import org.testng.annotations.Test;
 
+import static com.google.common.collect.ImmutableList.toImmutableList;
+import static java.util.concurrent.TimeUnit.MILLISECONDS;
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import static org.testng.Assert.assertTrue;
 
 public class TestTransactionManagerIntegration
 {
+    private static final Logger log = Logger.get(TestTransactionManagerIntegration.class);
+
     @Test
     public void testTransactionMetadataCleanup()
             throws Exception
     {
         try (DistributedQueryRunner queryRunner = TpchQueryRunnerBuilder.builder().build()) {
+            log.info("Start running testTransactionMetadataCleanup");
```

Comment:
do we still need this logging?

Reply:
I am not sure whether this PR will completely solve the issue,  so additional logging of remaining transaction ids is another purpose of this PR.

Reply:
that's covered in the assertTrue at line 41.  this logging just shows the beginning and end of the test.

Reply:
Right,  assertTrue will show up in error message, then I need to search previous logging to match which queries are not cleaned. An example of 5 actual transaction ids are shown here. https://github.com/prestodb/presto/issues/14835

Reply:
Got it. Can you create an issue to clean up the logging in e.g. 2 months if the error doesn't recur?